app/controllers/application_controller.rb

```diff
@@ -93,8 +93,7 @@ def check_hub_ready_for_checkout
     if current_distributor_closed?
       current_order.empty!
       current_order.set_distribution! nil, nil
-      flash[:info] = "The hub you have selected is temporarily closed for orders. "\
-        "Please try again later."
+      flash[:info] = I18n.t('check_hub_ready_for_checkout')
```

Comment:
this key is not well named. the key should describe the text, not the method where it appears.
maybe 'order_cycles_closed_for_hub' ?

Reply:
Yes, it's better. I will change.